content/docs/reference/crosswalk/aws/ecs.md

```diff
@@ -20,20 +20,20 @@ scale a cluster of virtual machines, or schedule containers on those virtual mac
 
 Pulumi Crosswalk for AWS ECS simplifies deploying containerized applications into ECS and managing all of the
 associated resources. This includes simple support for load-balanced container services and one-off tasks, in addition
-to managing the clusters and associated scaling, network, and security policies. This includes ECS Fargate -- the
-simplest option, alleviating the need to manage the cluster's servers themselves -- in addition to ECS classic --
+to managing the clusters and associated scaling, network, and security policies. This includes ECS Fargate---the
+simplest option, alleviating the need to manage the cluster's servers themselves---in addition to ECS classic---
 providing full control over the underlying EC2 machine resources that power your cluster.
 
-> An alternative to ECS is Amazon's Elastic Kubernetes Service (EKS). Similar ECS, EKS lets you operate
+> An alternative to ECS is Amazon's Elastic Kubernetes Service (EKS). Similar to ECS, EKS lets you operate
 > containerized applications in a cluster. EKS tends to be more complex to provision and manage, but has
 > the added advantage of using the industry standard container orchestrator, Kubernetes, and therefore can help
-> with portability between clouds and on-premises configurations. Please see the
-> [Pulumi Crosswalk for AWS EKS documentation]({{< relref "eks.md" >}}) for more information about using EKS.
+> with portability between clouds and on-premise configurations. See
```

Comment:
Random aside, we're inconsistent today on using "on-premise(s)" vs. "self-hosted." I prefer using the latter, since the self-hosted option isn't necessarily just for on-premise behind the firewall scenarios, but can also be hosted within your own AWS, Azure, or GCP account.

Reply:
Self-hosted sounds like a good candidate for the glossary. Thanks for giving this a thorough review!